Take OCS version from registry image if passed (#1653)
red-hat-storage/ocs-ci: docs/usage.md

### Additional arguments:

* `--cluster-name <name>` - name of cluster (always required for deployment) must be 5-17 characters long.
* `--ocs-version` - version of OCS to be used (e.g. 4.2 or 4.3). If not
    specified, the default from ocs-ci will be used. If --ocs-registry-image passed

>     than this parameter is ignored and version is parsed from registry image.

Reviewer: ```suggestion
    then this parameter is ignored and the version is parsed from the registry image.
```

* `--upgrade-ocs-version` - version of OCS to be used for upgrade (e.g. 4.2 or 4.3). If not specified, the default from ocs-ci will be used.
* `--ocs-registry-image` - ocs registry image to be used for deployment
    (e.g quay.io/rhceph-dev/ocs-olm-operator:latest-4.2). In case this parameter

>     is passed it overwrite the --ocs-version.

Reviewer: ```suggestion
    is passed, the version is parsed from the registry image name and it overwrites any version passed in via --ocs-version.
```

* `--cluster-name <name>` - name of cluster (always required for deployment) must be 5-17 characters long.
* `--ocs-version` - version of OCS to be used (e.g. 4.2 or 4.3). If not
    specified, the default from ocs-ci will be used. If --ocs-registry-image passed
    then this parameter is ignored and the version is parsed from the registry image.
* `--upgrade-ocs-version` - version of OCS to be used for upgrade (e.g. 4.2 or 4.3). If not specified, the default from ocs-ci will be used.
* `--ocs-registry-image` - ocs registry image to be used for deployment
    (e.g quay.io/rhceph-dev/ocs-olm-operator:latest-4.2). In case this parameter

>     is passed, the registry image name and it overwrites any version passed in

Reviewer: ```suggestion
    is passed, the version is parsed from the registry image name and it overwrites any version passed in
```

This was missing a few words from the suggestion. Like this it is not a complete sentence. ;-)
